Formula/griddb-c-client.rb

```diff
@@ -0,0 +1,37 @@
+class GriddbCClient < Formula
+  desc "Library for GridDB C Client provides a C interface for GridDB"
+  homepage "https://github.com/griddb/c_client"
+  url "https://github.com/griddb/c_client/archive/v4.5.1.tar.gz"
+  sha256 "bf9eaca4df14bd3badc662dc3a6db5cdcae5b35e9e50b48427c9b9dd02bc116e"
+  license "Apache-2.0"
+
+  bottle :unneeded
+
+  depends_on "autoconf"
+  depends_on "automake"
+  depends_on "libtool"
+  depends_on "llvm"
```

Comment:
Should these not be build-only dependencies?

Reply:
And does it not work with the macOS default LLVM?